### DISCUSSION TITLE
Red LED

### DISCUSSION THREAD

**Discussion by Foeteler** [adonno/tagreader#244] [Q&A]
Is there a possibility to turn off the red LED of the Reader? Beacause I will use it in the babys room the red LED is too bright.
That would be awesome!

**Comment by cpttater** [comment]
I would like to see if this is possible too as I will be putting it in my child's room.

**Reply by adonno** [reply]
So only issue iss either removing the red LED or, opening the device and putting a piece of black electrical tape on the backside of the Lid. Some filemantes aren't dark enough and they let the light through
